manifests/server/config.pp

```diff
@@ -157,22 +157,24 @@
 
   # Generate a new CA and host cert if our host cert doesn't exist
   if $puppet::server::ca {
```

Comment:
@ekohl [I followed your suggestion](https://github.com/theforeman/puppet-puppet/pull/783/files#r607761344). However, if this is the case, shouldn't there just be a $ca in the manifests/init.pp? There is no $ca option in manifests/init.pp now.

I just need this turned off so this module can complete doing its thing without failing about preexisting certs.

Reply:
We do have `$server_ca` in `init.pp`

Reply:
Hi @ekohl ,

> We do have $server_ca in init.pp

You must be referring [to this](https://github.com/theforeman/puppet-puppet/blob/master/manifests/init.pp#L215). I did see this, but this parameter has no effect on [whether to generate a CA file](https://github.com/theforeman/puppet-puppet/blob/master/manifests/server/config.pp#L168).

> We have a creates. Why isn't it picking that up?

Where in the code is this? I did not find a `creates` parameter in the init.pp.